frontend/src/components/Header/components/Nav/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function Nav({ className, pathname }: Props): JSX.Element {
         <Link href={ROUTES.COLLECTIONS} passHref>
           <AnchorButton
             active={isRouteActive(pathname, ROUTES.COLLECTIONS)}
-            data-testid="collection-link"
+            data-testid="collections-link"
```

Comment:
Updating `data-testid` from `collection-link` to `collections-link`. `collection-link` is used elsewhere and could create a problem distinguishing between the two use cases (when they co-exist on the same rendered page).